### PR TITLE
fix: stock reconciliation list pulling item with 0 stock

### PR DESCRIPTION
**Pull request by hitenshKharva**
Ref: [Asana Task](https://app.asana.com/0/1199082161983365/1200026413350136)

https://user-images.githubusercontent.com/56474017/111294171-0e576600-8670-11eb-9ac1-b20c0a6181f0.mp4

### CODE REVIEW

File: erpnext/stock/doctype/stock_reconciliation/stock_reconciliation.py [hunk 1]
```diff
@@ -468,7 +468,7 @@ def get_items(warehouse, posting_date, posting_time, company):
 		stock_bal = get_stock_balance(d[0], d[2], posting_date, posting_time,
 			with_valuation_rate=True)
 
-		if frappe.db.get_value("Item", d[0], "disabled") == 0:
+		if (frappe.db.get_value("Item", d[0], "disabled") == 0) and stock_bal[0]:
```
Comment (rvpasari) [inline erpnext/stock/doctype/stock_reconciliation/stock_reconciliation.py]: check for greater than zero, in case of negative balances

**Review by rvpasari — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/stock_reconciliation/stock_reconciliation.py [hunk 2]
```diff
@@ -468,7 +468,7 @@ def get_items(warehouse, posting_date, posting_time, company):
 		stock_bal = get_stock_balance(d[0], d[2], posting_date, posting_time,
 			with_valuation_rate=True)
 
-		if frappe.db.get_value("Item", d[0], "disabled") == 0:
+		if (frappe.db.get_value("Item", d[0], "disabled") == 0) and (stock_bal[0]>0):
```
Comment (vishdha) [inline erpnext/stock/doctype/stock_reconciliation/stock_reconciliation.py:471]:
```suggestion
		if (frappe.db.get_value("Item", d[0], "disabled") == 0) and (stock_bal[0] > 0):
```

**Review by vishdha — DISMISSED** [review]

File: erpnext/stock/doctype/stock_reconciliation/stock_reconciliation.py (hunk 2 above)
Comment (hrwX) [inline erpnext/stock/doctype/stock_reconciliation/stock_reconciliation.py]:
```suggestion
		if frappe.db.get_value("Item", d[0], "disabled") == 0 and stock_bal[0] > 0:
```

**Review by hrwX — CHANGES_REQUESTED** [review]

**Review by hrwX — DISMISSED** [review]

**Comment by rvpasari** [comment]
@sahil28297 don't merge this yet.

**Review by rvpasari — APPROVED** [review]

**Comment by rvpasari** [comment]
@sahil28297 resolved the queries I had related to this task, we can merge it now.

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**